main.tf

```diff
@@ -8,6 +9,14 @@ locals {
   option_group           = var.engine != "postgres" ? coalesce(module.db_option_group.this_db_option_group_id, var.option_group_name) : null
 }
 
+# Random string to use as master password
+resource "random_password" "master_password" {
+  count = var.create_db_instance && var.create_random_password ? 1 : 0
+
+  length  = 10
```

Comment:
I see the Aurora module has this hardcoded at 10, just feels like it might be worthwhile to have a `var.random_password_length` with a default value of 10 - what do you think?

Reply:
done